processing/src/main/java/io/druid/segment/Rowboat.java

```diff
@@ -127,9 +133,21 @@ public int compareTo(Rowboat rhs)
   @Override
   public String toString()
   {
+    final List<String> dimStrs = dims == null ? ImmutableList.<String>of() : Lists.transform(
```

Comment:
why not print what dims actually is when it's null? The point of toString here is not to be pretty but to make debugging easier, so we should print what the actual fields are.